Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openshift/origin-base
+FROM openshift/origin-release:golang-1.10 AS builder
```

Comment:
I've been using the full path in other images
registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
But if it's fine to use just the relative path, that's probably better anyway.